products/pluto/pluto_build/sql/zoning_specialdistrict.sql

```diff
@@ -3,10 +3,8 @@
 -- the order special purpose districts are assigned is based on which district covers the majority of the lot
 -- a district is only assigned if more than 10% of the district covers the lot
```

Comment:
@fvankrieken is this comment accurate? (or was it ever?) Isn't is the other way around? ie 10% of the lot must be covered by the district. Just checking my understanding.

Reply:
You are correct! Missworded comment